doc/Language/variables.pod6

```diff
@@ -328,7 +327,7 @@ to that block. This is because the placeholder variables are sorted in Unicode
 order. If you have self-declared a parameter using C<$^a> once, you may refer
 to it using only C<$a> thereafter.
 
-Although it is possible to use nearly any valid identifier as a placeholder
+Although it's possible to use nearly any valid identifier as a placeholder
```

Comment:
I don't like this contraction.  The one later on in the sentence is okay though. This is because you are not referring to something previously referenced using the word “it”, you are using it to refer to something later on in the same sentence (and it's also not at the beginning of a sentence).